torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1562,8 +1562,9 @@ Tensor renorm_backward(
   std::iota(reduce_dims.begin(), reduce_dims.end(), 0);
   reduce_dims.erase(reduce_dims.begin() + dim);
 
-  auto acc_type =
-      at::toAccumulateType(self.scalar_type(), /*is_cuda=*/self.is_cuda());
+  auto acc_type = self.is_mps()
+      ? self.scalar_type()
+      : at::toAccumulateType(self.scalar_type(), /*is_cuda=*/self.is_cuda());
```

Comment:
I wonder if we should move toAccumulateType() to have a `is_gpu` (or device) flag to be able to treat the accumulation type the same way on cuda and mps.

Reply:
yeah I thought about this but was not sure whether the cuda accumulation types are applicable to mps or not. So didn't change this part.

Will investigate it.